tests/performance/conftest.py

```diff
@@ -19,7 +21,9 @@ def pytest_addoption(parser):
         default=0.1,
         help="Percentage of links with same letters, eg: 0.1",
     )
-    parser.addoption("--seed", default=11, help="Randon number seed")
+
+    # TODO review bounds
```

Comment:
There is still a TODO in here, will it be addressed in another issue?

Reply:
It's OK if not, but please create a card for any TODO left in the code. Actually this is a general guideline: every TODO in code is supposed to be added as a card as well.

Reply:
Issue created https://github.com/singnet/das-query-engine/issues/353